mlos_bench/mlos_bench/tests/services/remote/azure/azure_services_test.py

```diff
@@ -151,7 +151,7 @@ def test_remote_exec_status(mock_requests: MagicMock, azure_vm_service: AzureVMS
     mock_response.status_code = http_status_code
     mock_requests.post.return_value = mock_response
 
-    status, _ = azure_vm_service.remote_exec(script, config={}, env_params={})
+    status, _ = azure_vm_service.remote_exec(script, config={"vmName": "test-vm"}, env_params={})
```

Comment:
Needs to be added since we don't (can't) include it as a part of the service config anymore.